### PR TITLE
Add documentation of packaging

### PR DESCRIPTION
**Pull request by cornelius**

### CODE REVIEW

File: docs/Packaging.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Packaging
+
+For installation on user's machines Machinery is packaged as gem and as RPM. The
```
Comment (mschnitzer) [inline docs/Packaging.md]: Shouldn't it be `GEM` rather than `gem` ?

File: docs/Packaging.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+systems, which have Ruby installed.
+
+The gem is published on RubyGems as
+[machinery-tool](https://rubygems.org/gems/machinery-tool). It includes the
+sources of the machinery helper, which are compiled on installation of the gem
+if there is a Go toolchain available.
+
+The RPM is maintained in the Open Build Service in the
+[systemsmamnagement:machinery](https://build.opensuse.org/project/show/systemsmanagement:machinery)
+project. The RPM bundles the gems it needs as well as the helper binary, so that
```
Comment (mschnitzer) [inline docs/Packaging.md]: I would say `The needed gems` or `The required gems`

File: docs/Packaging.md (hunk 1 above)
Comment (cornelius) [inline docs/Packaging.md]: "gem" is not an acronym, so it should be ok as it is.

**Comment by mschnitzer** [comment]
lgtm

**Pull request merged**